test/map/MapProxyTest.js

```diff
@@ -113,20 +113,20 @@ describe('MapProxyTest', function () {
             });
 
             it('put with ttl puts value to map', function () {
-                return map.put('key-with-ttl', 'val-with-ttl', 3000).then(function () {
+                return map.put('key-with-ttl', 'val-with-ttl', 20000).then(function () {
                     return map.get('key-with-ttl').then(function (val) {
                         return expect(val).to.equal('val-with-ttl');
                     });
                 });
             });
 
             it('put with ttl removes value after ttl', function () {
-                return map.put('key10', 'val10', 1000).then(function () {
+                return map.put('key10', 'val10', 20000).then(function () {
```

Comment:
I think it is safe to use a shorter timeout for this test. We already test the put and get scenario on `put with ttl puts value to map`.  I believe it would be OK to make the TTL shorter and get rid of the map.get assertion

Reply:
Or, get rid of the `put with ttl puts value to map` altogether, and just have this test, which is similar to others

Reply:
Done. Forwardporting it to other branches